ext/openssl/php_openssl.h

```diff
@@ -86,6 +94,13 @@ php_stream_transport_factory_func php_openssl_ssl_socket_factory;
 
 void php_openssl_store_errors();
 
+PHP_OPENSSL_API int php_openssl_cipher_iv_length(char *method, zend_long *ret);
+PHP_OPENSSL_API int php_openssl_random_pseudo_bytes(zend_string **buffer, zend_long length);
+PHP_OPENSSL_API int php_openssl_encrypt(char *data, size_t data_len, char *method, size_t method_len, char *password,
+		size_t password_len, zend_long options, char *iv, size_t iv_len, zval *tag, zend_long tag_len, char *aad, size_t add_len, zend_string **outbuf);
```

Comment:
Could you pls wrap this line as well. It's great if I can see the whole line in here without side scrolling... :)